web/src/main/java/ca/hendriks/bartender/web/ingredient/IngredientsRestController.java

```diff
@@ -18,14 +18,26 @@ public IngredientsRestController(final IngredientRepository ingredientRepository
     }
 
     @GetMapping
+    @RequestMapping("/ingredients")
```

Comment:
redundant. The `@RequestMapping` annotation has already set the same path for the whole class.